parts/linux/cloud-init/artifacts/labels.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+# Update Labels for Kubernetes nodes
+
+set -euo pipefail
+
+for kubelet_label in $(echo $KUBELET_NODE_LABELS | sed "s/,/ /g")
+do
+  kubectl label --kubeconfig /var/lib/kubelet/kubeconfig --overwrite nodes $HOSTNAME $kubelet_label
```

Comment:
We should lowercase the hostname

Reply:
The uppercase hostname works. Is there a particular reason we should use the lowercase one?

Reply:
Because kubelet lower-cased the hostname https://github.com/kubernetes/kubernetes/blob/master/pkg/util/node/node.go#L53.
Your code is working because we now have hostnames lower-cased. 
CC @xuto2

Reply:
My mind for some reason was blanking on the fact that ```hostname``` was a command 🙈
 
I was thinking of them most as envvars, but didn't get why hostname as the envvar when I didn't spot it in the env and thought it was strange being lower case.
 
Since we use $(hostname) elsewhere in agentbaker I think you are right that it makes sense to use that.

Reply:
I think there is a misunderstanding but @mainred  is right. 
L69 of the node.go intentionally make the node name a lower case string, regardless the value of the hostname, so better to do the same here.
https://github.com/kubernetes/kubernetes/blob/3667e0e9f7cea2a36a36f37cd8dfdb6eeca7232c/pkg/util/node/node.go#L69
return strings.ToLower(hostName), nil

@charliedmcb  you were talking about hostname vs HOSTNAME, which is a different thing, but also we should use hostname instead of HOSTNAME (it won't work)

Reply:
Should have updated this with our offline conversation. 

I realized we were talking about two different things. 

I will switch it to using ```$(hostname)``` and convert to lowercase

However, @xuto2 I'm still curious why HOSTNAME would not work? Its been working in all the tests I've run.

Reply:
@xuto2 I'm still curious why HOSTNAME would not work?

Reply:
Still curious, but resolving